docs/portal/image-registry/updating-projects.md

```diff
@@ -0,0 +1,15 @@
+# Updating Projects
```

Comment:
Again, the contents and organization on this page are a bit inconsistent with previous pages. Please refer to this page "updating-image-pull-secret" and try to maintain the consistency.

Note: the above applies to "Updating Users" page as well.

Reply:
Done.